ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -122,20 +131,70 @@ const getDescription = (
   }
 };
 
+const isTransactionMeta = (
+  confirmation: Confirmation | undefined,
+): confirmation is TransactionMeta => {
+  return (
+    confirmation !== undefined &&
+    (confirmation as TransactionMeta).txParams !== undefined
+  );
+};
```

Comment:
`useConfirnContext< TransactionMeta >` can be used to avoid casting.

Reply:
Current confirmation can also be a signature, so I don't think we can do that in this case.